.npmignore

```diff
@@ -1,31 +0,0 @@
-# dependencies
```

Comment:
.npmignore is still present to prevent npm from using .gitignore (default behavior) -- but using whitelisting now instead so the file is just a placeholder